samples/bluetooth/nrf_dm/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -7,3 +7,15 @@
 &ipc0 {
 	zephyr,buffer-size = <1536>;
 };
+
+ /* Enable pin forwarding to network core. The selected pins will be used by
+  * Distance Measurement module for debug purposes.
+  *
+  * Note: Pay attention to assign the same GPIO pins as Kconfig file.
+  */
+&gpio_fwd {
```

Comment:
Do you want to always forward these pins?

Reply:
CONFIG_DM_GPIO_DEBUG is enabled in the sample, so we forward the pins.